lib/Core/Executor.cpp

```diff
@@ -1343,6 +1347,15 @@ Executor::toConstant(ExecutionState &state,
   return value;
 }
 
+ref<klee::ConstantExpr>
+Executor::getValueFromSeeds(std::vector<SeedInfo> &seeds, ref<Expr> e) {
+  assert(!seeds.empty());
+  auto value = seeds.back().assignment.evaluate(e);
```

Comment:
There are two issues in this line:
* `evaluate` can return a non-constant expression
* checking `back` is not enough. All the entries in `seeds` have to be checked if any of them returns a `ConstantExpr`. This could fail if it handles addresses that don't match anymore on reruns.

Reply:
Done.